backoffice/src/Models/Enums/RecoveredFrom.cs

```diff
@@ -0,0 +1,9 @@
+namespace Droits.Models.Enums;
+
+public enum RecoveredFrom
+{
+    Shipwreck,
```

Comment:
Make sure to have the nicely displayed ones (For when it comes in the dropdown with spaces etc)